tests/phpunit/integration/tests/Experiments.php

```diff
@@ -84,11 +84,14 @@ public function test_add_menu_page() {
 	 * @covers ::initialize_settings
 	 */
 	public function test_initialize_settings() {
+		global $wp_settings_fields, $wp_settings_sections;
+
 		$experiments = new \Google\Web_Stories\Experiments();
 		$experiments->initialize_settings();
 
-		$options = get_registered_settings();
```

Comment:
Why can this change?

Reply:
`get_registered_settings` checks which settings are registered.

Setting registration is the job of the `Settings` class, not the `Experiments` class.

Here we are only concerned about testing the `Experiments` class though.